components/email/src/Email.svelte

```diff
@@ -408,16 +408,15 @@
     }
     loading = true;
     if (_this.click_action === "default" || _this.click_action === "mailbox") {
-      //#region read/unread
-      if (
-        activeThread &&
-        activeThread.unread &&
-        _this.click_action !== "mailbox"
-      ) {
-        activeThread.unread = !activeThread.unread;
-        await saveActiveThread();
+      // default click action
+      if (activeThread && _this.click_action === "default") {
+        // Setting read/unread status
+        if (activeThread.unread) {
```

Comment:
```suggestion
        if (activeThread?.unread) {
```

Reply:
@pooja169usp Isn't it guarded with the previous if statement? `if (activeThread && _this.click_action === "default")`